src/test/common/Throttle.cc

```diff
@@ -357,15 +357,15 @@ TEST(BackoffThrottle, destruct) {
       ASSERT_TRUE(throttle->set_params(0.4, 0.6, 1000, 2, 10, 6, nullptr));
 
       throttle->get(5);
-      {
-	auto& t = *throttle;
-	std::thread([&t]() {
-	    usleep(5);
-	    t.get(6);
-	  });
-      }
+      auto& t = *throttle;
+      std::thread runner([&t]() {
+	  usleep(5);
+	  t.get(6);
+	});
       // No equivalent of get_or_fail()
       std::this_thread::sleep_for(std::chrono::milliseconds(250));
+      throttle.reset();
```

Comment:
BackoffThrottle doesn't not have reset() member function.

Reply:
`unique_ptr` does.

Reply:
learned